lazyflow/operatorWrapper.py

```diff
@@ -90,12 +84,9 @@ def __init__(
 
         self._initialized = False
 
-        if operatorClass.name == "":
-            self._name = "Wrapped " + operatorClass.__name__
-        else:
-            self._name = "Wrapped " + operatorClass.name
-
-        self._customName = False
+        self.name = "Wrapped " + operatorClass.name
+        if self._debug_logger:
+            self._debug_logger.debug(f"Wrapper {id(self)} name={self.name}")
```

Comment:
I don't think there is a way currently to get the operatorwrapper a debug logger (_write_logs not exposed). Same goes for any other operator class that overrides `__init__` (e.g. `OpReorderAxes`). And in general I think more operators should use init to set initial values. So currently the following operators will not be debuggable like this (because they override `__init__` without passing `**kwargs` up:

* `OperatorWrapper`
* `OpPixelOperator`
* `OpReorderAxes`
* `OpObjectFeatures`
* `Workflow`
* `OpManualTracking`
* `OpDifference`
* `OpZeroBasedConsecutiveIndexRelabeling`
* `OpRelabeledMergerFeatureExtraction`
* `OpConservationTracking`
* `OpTrackingFeatureExtraction`
* `OpAnnotations`
* `OpCarving`

Reply:
I see you mention this, too, in your last comment.